skribenten-backend/.nais/dev.yaml

```diff
@@ -22,6 +22,8 @@ outboundRules:
   - application: pensjonsbrev-tjenestebuss-q2
     namespace: pensjonsbrev
     cluster: dev-fss
+  - application: navansatt
+    namespace: teampensjon
```

Comment:
Skrive hvilket cluster også for å være likt med de andre? Trengs nok ikke, men hjelper for å fort se hvor tjenesten ligger